### ISSUE TITLE
Features of your plugin

### ISSUE THREAD

**Issue by Free-IPTV** [PseudoTV/PseudoTV_Live#29]
Hi Lunatixz ,

I am looking for an alternative for the IPTV-MERGE. Is it possible to use multiple m3u's and multiple EPG sources with your addon and create a playlist from them as m3u for the IPTV Simple Client?

Greetings

Free-IPTV

**Comment by Lunatixz** [comment]
No, project supports importing a single m3u/xmltv pair. Users can merge multiple m3u's using one of the many tools available on the net.
